code-challenges.js

```diff
@@ -8,54 +8,185 @@
+//return: the total number of elements of merged arrays console.log(myNumbers1.length + myNumbers2.length)
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+console.log(myNumbers1.length + myNumbers2.length)
+
+//Output: 10
+
+//I honestly did not think that would work. I've told myself to stop over thinking and try the simplest way to work the problem first then start branching out.
+
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 atleD"
 
 // Pseudo code:
 
+//problem: the letters in the string need to be reversed
+//function expression: I'm going to try .reverse() to see if that works with the string. If not then I'll try charsReversed.
+//set parameters: the letters in the string need to be be in reverse order
+//make decision: using .reverse() should work to return the letters in reverse order 
+//return: the letters in the string to be reversed. 
+    //After trying both of those and doing more research on w3 and the syllabus, I've decided to try indexing the string and then reversing it.
+    //That did not work so I will try joining and then reversing the string.
+    //Also did not work so I will do the opposite of join and split the string.
+
 const currentCohort = "Delta 2022"
+console.log(currentCohort.split(""))
+
+var cohortSplit = currentCohort.split("")
+    console.log(cohortSplit.reverse(), cohortSplit.join(""))
+return(cohortSplit.reverse(), cohortSplit.join(""))
+//Output: 2202 atleD
+
+//This took me about an hour and a half to figure it out. I figured out I had to split and join and reverse the values. I went back to the syllabus to see which order to put it in and also went back to the practice we did in class. It was pretty frustrating that the code didn't work like how i thought it was going to the first time.
+
 
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
 // Pseudo code:
-
-const myArray = [13, 34, 5, 10, 27, 42]
+//problem: the numbers in the array need to be identified as odd or even and returned as such.
+//function expression: using a for loop, index the list of numbers and then have the new indexed array % 2 === 0 for the even numbers. For the odd numbers, the indexed array $ 2 == 1. 
+//set parameters: myArray[i] % 2 === 0; myArray[i] % 2 == 1. 
+//make decision: write a for loop indexing starting with i=0, ending at i<=42 (largest number in the array). Then write an if for the odd numbers and another if function for the even numbers. If the number is odd or even return the appropriate word ("even" or "odd").
+//return: "even" "odd"
+
+const myArray = [13, 34, 5, 10, 27, 42] 
+for(let i=0; i<=42; i++){
```

Comment:
Careful here in your for loop where you have i <= 42.  You would usually want to compare to the length of the array as that will be your stopping point.